frontend/src/pages/TermPlanner/main.less

```diff
@@ -206,8 +208,9 @@ body {
     image-rendering: auto;
     border-radius: 12px;
     cursor: help;
+    transition: box-shadow 0.5s;
```

Comment:
could make the speed 0.3s instead? Feels like the hover animation is a bit slow on 0.5s. Also, good to specify the transition effect for readability. So `transition: box-shadow 0.3s ease` would work!

Reply:
oo sounds good :)